ui-tests/truffle/truffle-config.js

```diff
@@ -1,4 +1,8 @@
-// const path = require("path");
+const HDWalletProvider = require('@truffle/hdwallet-provider')
+
+const mnemonic = process.env['MNEMONIC']
+const rinkeby_endpoint = process.env['RINKEBY_ENDPOINT']
```

Comment:
Not going to fight this but how do you feel about not using snake case for this and using camel case instead?

Reply:
a good argument